modP2p/src/org/aion/p2p/P2pConstant.java

```diff
@@ -1,3 +1,28 @@
+/*
+ * Copyright (c) 2017-2018 Aion foundation.
+ *
+ * This file is part of the aion network project.
+ *
+ * The aion network project is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * The aion network project is distributed in the hope that it will
+ * be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the aion network project source files.
+ * If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors to the aion source files in decreasing order of code volume:
+ *
+ * Aion foundation.
+ *
+ */
+
 package org.aion.p2p;
```

Comment:
these license headers look like an old version, please update to the [new standard](https://github.com/aionnetwork/aion/wiki/Aion-License-Header)

Reply:
updated in https://github.com/aionnetwork/aion/pull/588/commits/49f9d0436bbce7fdda0bc17dc88d4c170f67eff8